src/main/java/com/example/booksstore/service/impl/OrderItemServiceImpl.java

```diff
@@ -25,7 +25,7 @@ public class OrderItemServiceImpl implements OrderItemService {
     @Override
     public List<OrderItemResponseDto> getAllOrderItemsByOrderId(Long orderId) {
         orderRepository.findById(orderId).orElseThrow(() ->
-                new NoSuchOrderException("Can't find order with id " + orderId));
+                new NoSuchOrderException("Can't find order, it does not belong to you."));
         return orderItemRepository.findByOrderId(orderId).stream()
                 .map(orderItemMapper::toDto)
                 .toList();
```

Comment:
You have changed the message but I do not see a validation, looks like user now can send some request with order id of some other user, and that would be insecure, we should fix that and check if the order that we have fetched actually belongs to that user who is making the request